docs/testing/sideload-office-add-ins-for-testing.md

```diff
@@ -65,12 +65,16 @@ The following video walks you through the process of sideloading your add-in on
 
 ## Sideload an add-in when using Visual Studio
 
-If you're using Visual Studio to develop your add-in, the process to sideload is similar. The only difference is that you will have to update the value of the **SourceURL** element in your manifest to include the full URL where the add-in is deployed. 
+If you're using Visual Studio to develop your add-in, the process to sideload is similar. The only difference is that you must update the value of the **SourceURL** element in your manifest to include the full URL where the add-in is deployed.
 
-If you're currently developing your add-in, locate your add-in manifest.xml file, and update the **SourceLocation** element value to include an absolute URI. Visual Studio will put in a token for your localhost deployment.
-
-For example: 
-
-```xml
-<SourceLocation DefaultValue="https://localhost:44300/App/Home/Home.html" />
-```
+> [!NOTE]
+> Although you can sideload add-ins from Visual Studio to Office Online, you cannot debug them from Visual Studio. To debug you will need to use the browser debugging tools. For more information, see [Debug add-ins in Office Online](debug-add-ins-in-office-online.md).
+
+1. In Visual Studio, show the **Properties** window by choosing **View** -> **Properties Window**.
+2. In the **Solution Explorer**, select the web project. This will display properties for the project in the **Properties** window.
+3. In the Properties window, copy the **SSL URL**.
+4. In the add-in project, open the manifest XML file. Be sure you are editing the source XML. For some project types Visual Studio will open a visual view of the XML which will not work for the next step.
+5. Search and replace all instances of "~remoteAppUrl/" with the SSL URL you just copied. You will see several replacements depending on the project type, and the new URLs will appear similar to 'https://localhost:44300/Home.html'.
```

Comment:
Sorry, two more minor changes to this step:

- remove quotes around "~remoteAppUrl/" and instead format as bold:  **~remoteAppUrl/**  -- to avoid any potential confusion around whether or not the quotes are part of the string they're searching for.

- the single quotes around the URL at the end of this step should be back-ticks \`url_here\`

And as long as you're making these changes, please update the phrase "on either Office 365 or Office Online" in the opening paragraph of the article to "in either Office 365 or Office Online" :)